de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/toolbar/ToolbarViewPresenterImpl.java

```diff
@@ -394,4 +410,63 @@ public void onSuccess(String result) {
         });
     }
 
+    @Override
+    public void onAutomateHTPathlist() {
+        drFacade.getInfoTypes(new DataCallback<List<InfoType>>() {
```

Comment:
I haven't gotten to writing test yet. I wanted to get PR out in time for Monday drop